HelloWorld_ST_Sensors/user/bt/hc05.cpp

```diff
@@ -0,0 +1,32 @@
+#include "hc05.hpp"
+
+namespace bt {
+
+void hc05::write(const char* data) {
```

Comment:
Since we are already using class based functionality, I did not make the functions verbose that way
For example

```cpp
bt::hc05 btserial(parameters);
btserial.write("My data");
```
As you can see the code is very readable for the user

Reply:
NOTE: Also updated the API
```cpp
char buffer[] = "Hello World";
btserial.write(buffer, sizeof(buffer));
```